src/averageAges.js

```diff
@@ -15,11 +15,17 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let filtered = people.filter(el => el.sex === 'm');
+
+  if (arguments.length > 1) {
+    filtered = people
```

Comment:
replace `if ()` statement with &&, || or ?:
and check the same rule in whole document